### PR TITLE
Ux study

### PR DESCRIPTION
**Pull request by JoshLove-msft**
- Removes samples from README
- Adds send IEnumerable
- Adds ServiceBusSenderOptions

### CODE REVIEW

File: sdk/servicebus/Azure.Messaging.ServiceBus/README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ To interact with these resources, one should be familiar with the following SDK
 
 - A **Service Bus sender** is scoped to a particular queue or topic, and is created using the Service Bus client. The sender allows you to send messages to a queue or topic. It also allows for scheduling messages to be available for delivery at a specified date.
 
-- A **Service Bus receiver** is scoped to a particular queue or subscription, and is created using the Service Bus client. The receiver allows you to receive messages from a queue or subscription. It also allows the messages to be settled. There are four ways of  settling messages:
+- A **Service Bus receiver** is scoped to a particular queue or subscription, and is created using the Service Bus client. The receiver allows you to receive messages from a queue or subscription. It also provides provides the following methods of settling a message:
```
[the remark this answers is not in the document]
Comment (ShivangiReja) [reply sdk/servicebus/Azure.Messaging.ServiceBus/README.md]: Can we also mention something like - these are the message settlement methods you would call after receiving messages?

**Comment by ShivangiReja** [comment]
@JoshLove-msft should we remove the creation of client and creation of processor from README? I think adding handlers, starting and stopping processor would be enough.

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusSender.cs [hunk 2]
```diff
@@ -236,15 +236,16 @@ protected ServiceBusSender()
         }
 
         /// <summary>
-        ///   Sends a set of messages to the associated Service Bus entity using a batched approach.  If the size of messages exceed the
-        ///   maximum size of a single batch, an exception will be triggered and the send will fail.
+        ///   Sends a <see cref="ServiceBusMessageBatch"/> to the associated Service Bus entity using
+        ///   a batched approach.  If the size of messages exceed the maximum size of a single batch,
+        ///   an exception will be triggered and the send will fail.
```
[the remark this answers is not in the document]
Comment (JoshLove-msft) [reply sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusSender.cs:241]: Good point.

**Comment by JoshLove-msft** [comment]
> @JoshLove-msft should we remove the creation of client and creation of processor from README? I think adding handlers, starting and stopping processor would be enough.

Creation of client was intentionally left in. I think creation of processor could be removed.

**Pull request closed without merging**